src/commands/commit/README.md

```diff
@@ -0,0 +1,49 @@
+# Commit command
+
+Upload the git commit details to Datadog.
+
+## Usage
+
+### Setup
+
+You need to have `DATADOG_API_KEY` in your environment.
+
+```bash
+# Environment setup
+export DATADOG_API_KEY="<API KEY>"
+```
+
+It is possible to configure the tool to use Datadog EU by defining the `DATADOG_SITE` environment variable to `datadoghq.eu`. By defaut the requests are sent to Datadog US. In order to send the usage metrics to the correct datacenter, the `DATADOG_API_HOST` must also be set if another instance of Datadog is used. For example, for Datadog EU, it should be set to `api.datadoghq.eu`.
```

Comment:
Will the setup of sending git commits details to one site and usage metrics to another one be common?
If not, could we infer the API host from the `DATADOG_SITE` so changing site is only one variable away?

(Also it seems the `DATADOG_SOURCEMAP_INTAKE_URL` variable is mostly internal and will not be used by customers so maybe should not be documented here?)

Reply:
I don't think it's common but i'll ask to double check just in case.

Reply:
It's not common indeed, and doing so should probably not be documented. I didn't look at the code for this new command yet, but this is already what is done for sourcemaps upload (inferring both srcmap intake and metrics intake url from `DATADOG_SITE`).

Reply:
Indeed it's inferred in both cases.
The README from the sourcemaps upload command that was copied over was outdated.
This PR updates it: https://github.com/DataDog/datadog-ci/pull/284